jira.sgnl.yaml

```diff
@@ -0,0 +1,149 @@
+# Jira Configuration YAML for DS2.0
+displayName: "Jira"
+icon: ""
+description: "Jira as a System of Record"
+address: "{{Input Required: Jira-Domain}}.atlassian.net"
+defaultSyncFrequency: HOURLY
+defaultSyncMinInterval: 1
+defaultApiCallFrequency: SECONDLY
+defaultApiCallMinInterval: 1
+adapter:
+  hostname: "adapter-sgnl.sgnl.svc.cluster.local"
+  port: 80
+  config: "{{Input Required: Config}}" # Empty config: "e30="
```

Comment:
Specifying an empty config `""` did not work on the FE for me. It must be specified with an empty `{}` b64 encoded.

Reply:
The config doesn't need to be empty for the adapter to work. It works if it's empty or not. Sorry for the confusion! I mean't that if you want to specify an empty config, you must specify `"e30="` as opposed to just leaving it blank `""`. The reason for that is because if I specify `config = ""`, the `Continue` button is grayed out: 
<img width="1153" alt="image" src="https://github.com/SGNL-ai/catalog-sgnl-templates/assets/39134434/7f4b2d01-1614-45bc-ac9b-3674575cc4e8">


All the Jira adapter needs to be able to sync is the correct address, username, and password (API token).

We are only using `config` to specify the `issuesJqlFilter`, and if the customer does not need to specify that filter, then the `config` can be left blank.

Should I add comments to clarify this? I suppose we'll include this in the Jira SGNL docs.

Reply:
This might be a bug on the FE? @ivanfong.

Also, let me try creating this DS using the API with the exact same template.

Reply:
@richardxgao I'm seeing the following error in the grafana logs:

```
{"level":"error","ts":"2023-08-09T06:58:15Z","msg":"Current page has exceeded maximum allowed retry attempts. Restarting full sync","workerId":"15e351a2-9b54-4a56-bf37-1560b2fba1e1","clientId":"bd107052-7a51-4358-9282-6fb62b73fec9","tenantId":"01f32892-4794-4801-b339-8ce4c5cf376d","datasourceName":"Jira","datasourceId":"ebd98d13-554b-4f46-9671-d50d4d3f1a41","entityId":"9e5b87a6-778a-4a25-83f2-cde088b70266","entityType":"JiraUser","pageNumber":1,"taskExecId":"700f48ac-3353-4bcc-94c5-1b8c0d36fd23","retryCount":5,"taskPageExecId":"9ce653d1-5a60-47d3-af0c-35c151871cee","retryCount":5,"eventType":"sgnl.ingestionSvc.failed","error":"Config in datasource config could not parsed as JSON: json: cannot unmarshal string into Go value of type jira_adapter.Config."}

```

I used base64 encoded "{}" as config.

Reply:
This was resolved over call. Transient error most likely.